docs/creating_integration_manifest.md

```diff
@@ -96,21 +96,21 @@ If your integration supports discovery via [Zeroconf](https://en.wikipedia.org/w
 
 ## SSDP
 
-If your integration supports discovery via [SSDP](https://en.wikipedia.org/wiki/Simple_Service_Discovery_Protocol), you can add the type to your manifest. If the user has the `ssdp` integration loaded, it will load the `ssdp` step of your integration's config flow when it is discovered. We support SSDP discovery by ST, manufacturer and device type. Your integration is discovered if any of the specified info is found. It's up to your config flow to filter out duplicates.
+If your integration supports discovery via [SSDP](https://en.wikipedia.org/wiki/Simple_Service_Discovery_Protocol), you can add the type to your manifest. If the user has the `ssdp` integration loaded, it will load the `ssdp` step of your integration's config flow when it is discovered. We support SSDP discovery by ST, and all data in UPnP device description. The manifest value is a list of matcher dictionaries, your integration is discovered if all items of any of the specified matchers are found in the SSDP/UPnP data. It's up to your config flow to filter out duplicates.
 
 ```json5
 {
-  "ssdp": {
+  "ssdp": [{
```

Comment:
Let's format the JSON a little more readable with the list start/end characters on different lines than the dict.

It feels weird that we now have a list of lists. I think that we should make the values of `st` etc now just strings.

Reply:
True that, will address soon.

Reply:
Done now.